Readme.md

```diff
@@ -32,6 +32,26 @@ it "sets $foo on commit" do
 end
 ```
 
+### Temporary disable after commit hooks
+
+In your test_helper, you can specify the default
+
+```
+TestAfterCommit.enabled = true
+```
+
+Then use blocks in your tests to change the behavior:
+
+```
+TestAfterCommit.with_commits(true) do
```

Comment:
I could use "enabled", but would have to rename the config/attr_accessor. They collide and you get interesting errors depending on the order you define them :)

Would you rather see this?
```
TestAfterCommit.enabled_by_default = false

TestAfterCommit.enabled(false) do
end
```